Makefile

```diff
@@ -111,7 +111,7 @@ vet: ## Run go vet against code.
 
 .PHONY: test
 test: codegen fmt vet envtest ## Run tests.
-	KUBEBUILDER_ASSETS="$(shell $(ENVTEST) use $(ENVTEST_K8S_VERSION) --bin-dir $(LOCALBIN) -p path)" go test -race -short -v $$(go list ./... | grep -v /tests/e2e) 
+	KUBEBUILDER_ASSETS="$(shell $(ENVTEST) use $(ENVTEST_K8S_VERSION) --bin-dir $(LOCALBIN) -p path)" go test -p 1 -race -short -v $$(go list ./... | grep -v /tests/e2e) 
```

Comment:
we can't run test suites in parallel - if we do, they stomp on each other

Reply:
I'm seeing ~10 minutes instead of ~8 minutes

Reply:
another option in the future would be to create parallelization through a matrix of tests that each test a subset of the tests.

We can explore how other repositories are dealing with this problem.

Reply:
Actually, what I read is that if you don't use `-p 1`, then the default behavior is that "multiple test suites" run in parallel. I think other repos aren't using test suites at all, so I believe that their unit tests run sequentially. Just ours is using the Ginkgo test suite.